alana/prompt.py

```diff
@@ -182,7 +188,8 @@ def gen(
 
 
 def gen_msg(
-    messages: List[MessageParam],
+    messages: Optional[List[MessageParam]] = None,
+    user: Optional[str] = None,
```

Comment:
It's `user` to match the rest of the API.

I see how it could be confusing, but seems like a big change to move away from this.

Reply:
If you are using PyCharm you can just press `Ctrl+Shift+R` to replace it everywhere at once.

I'm not pushing but I think that this should be change since it's a variable which end users are/will be using so it should be clear what it's representing. If not now I would for sure place it on TODO list.